src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java

```diff
@@ -122,44 +129,53 @@ public void deleteInstanceThrowsRequestExceptionTest()
 
         this.openStackAttachmentPlugin.deleteInstance(instanceId, this.localToken);
     }
-
+    
+    // FIXME: is this test useful?
+    //test case: Check if a attachment returned by getInstance is not null when a valid instance is returned by getInstanceFromJson.
```

Comment:
I think it is better to improve the test by assert the contents of attachmentInstance. I suggest you to delete the text and add a TODO mark indicating that.

Reply:
The test was properly rewritten.